### PR TITLE
Update cancun evm changes and revise hardfork contents

### PR DESCRIPTION
**Pull request by yoomee1313**
## Proposed changes
This PR updates changes to Cancun EVM. Updating those changes was difficult within the current structure, the structure is slightly modified before applying the changes.
- Remove hardfork-previous pages. Now, hardfork changes are listedat the bottom of related pages.
- Grouped gas-related content under transaction fees.
  - transaction-fees
    - intrinsic-gas
    - execution-gas

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/learn/computation/precompiled-contracts.md [hunk 1]
```diff
@@ -1,32 +1,18 @@
...
 
 Klaytn provides several useful precompiled contracts. 
-These contracts are implemented in the platform itself as a native implementation. 
-The precompiled contracts from address 0x01 through 0x09 are the same as those in Ethereum. 
-Klaytn additionally implements precompiled contracts from 0x3fd through 0x3ff to support new Klaytn features.
+These contracts are implemented in the platform itself as a native implementation.
+The precompiled contracts from address 0x01 through 0x0A are the same as those in Ethereum. 
+Klaytn additionally implements precompiled contracts from 0x3FD through 0x3FF to support new Klaytn features.
 
 :::note
```
Comment (scott-klaytn) [inline docs/learn/computation/precompiled-contracts.md:8]: Same as before. An empty line after ":::note" and before ":::".

File: docs/learn/transaction-fees/execution-gas.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# Execution Gas
+
+Gas is a sum of `IntrinsicGas` and `ExecutionGas`. In here, we would focus on how `ExecutionGas` is calculated.
+:::note
```
Comment (scott-klaytn) [inline docs/learn/transaction-fees/execution-gas.md:4]: Be sure to add an empty line before and after ":::note" and ":::".

File: docs/learn/transaction-fees/intrinsic-gas.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Intrinsic Gas
+
+Gas is a sum of `IntrinsicGas` and `ExecutionGas`. In here, we would focus on how `IntrinsicGas` is organized.
+
+:::note
```
Comment (scott-klaytn) [inline docs/learn/transaction-fees/intrinsic-gas.md:5]: Same syntax correction needed for ":::note".

File: docs/learn/why-klaytn.md [hunk 4]
```diff
@@ -77,7 +77,7 @@ Klaytn does not sacrifice blockchain’s core characteristics to achieve the abo
 [Human-Readable Address]: ./accounts.md#human-readable-address-hra
 [Consensus Mechanism]: ./consensus-mechanism.md
 [Affordable Smart Contract Execution Cost]: computation/klaytn-smart-contract.md#affordable-smart-contract-execution-cost
-[Transaction Fees]: ./transaction-fees.md
+[Transaction Fees]: transaction-fees/transaction-fees.md
```
Comment (scott-klaytn) [inline docs/learn/why-klaytn.md:80]: Can you please check other instances of the link that refers to "transaction-fee.md"? If you've already done with this, ignore my comment.

File: docs/learn/computation/execution-model.md [hunk 5]
```diff
@@ -74,7 +74,10 @@ A _smart contract_ consists of a collection of code \(functions\) and data \(sta
...
+A smart contract can be created in the Klaytn blockchain by sending a transaction to an empty address with the binary as data. The binary can be in various formats; however, Klaytn currently supports one binary format, EVM bytecode. It is worth pointing out that this transaction requires a payment for execution. The account balance on the sender's account will be reduced according to the transaction fee model after the transaction has been stored in a block. After some time, the transaction should appear in a block, which confirms that the state it entails reached a consensus. At this point, the smart contract now exists in the Klaytn blockchain. 
+- As [eip-3541](https://eips.ethereum.org/EIPS/eip-3541) is brought at the Kore hardfork, deployment of a new code starting with the 0xEF byte is not allowed.
+- As [eip-3860](https://eips.ethereum.org/EIPS/eip-3541) is brought at the Shanghai hardfork, deployment of a new code is rejected if the length exceeds 49152 bytes and the length of the new code cannot exceed 24576 bytes.
```
[the remark this answers is not in the document]
Comment (yoomee1313) [reply docs/learn/computation/execution-model.md]: Wow thank you.. Reflected at here https://github.com/klaytn/klaytn-docs/pull/655/commits/9013b9ea79ef566d4f698c636136c145e9b85a91.

File: docs/learn/accounts.md [hunk 6]
```diff
@@ -186,16 +186,9 @@ In order for a transaction to be valid for an account associated with AccountKey
 * The number of signed public keys should be less than the number of weightedPublicKeys.
 
 :::note
```
[the remark this answers is not in the document]
Comment (yoomee1313) [reply docs/learn/accounts.md:188]: Reflected at here https://github.com/klaytn/klaytn-docs/commit/9013b9ea79ef566d4f698c636136c145e9b85a91.

File: docs/learn/why-klaytn.md (hunk 4 above)
Comment (yoomee1313) [reply docs/learn/why-klaytn.md:80]: I have checked that the links that refers to "transaction-fee.md" in english docs are changed correctly, but in korean and vietnamese docs, they still refers "transaction-fee.md". I'm not sure I can change this. Can I change those links too?

Comment (scott-klaytn) [reply docs/learn/why-klaytn.md:80]: Whenever you change a link in English version, it's supposed to be reflected to other languages automatically when a pull request of localized versions from Crowdin caused by a change in English is approved. Thus, you don't need to care about the other languages when you change something in English unless you want to add or improve the translations in Crowdin.

**Review by scott-klaytn — APPROVED** [review]

File: docs/learn/why-klaytn.md (hunk 4 above)
Comment (yoomee1313) [reply docs/learn/why-klaytn.md:80]: @scott-klaytn That's clear.. thank you.

**Review by ayo-klaytn — APPROVED** [review]

**Review by blukat29 — APPROVED** [review]

**Pull request merged**